cmd/gazelle/diff_test.go

```diff
@@ -105,12 +105,17 @@ func TestDiffNew(t *testing.T) {
 func TestDiffEmpty(t *testing.T) {
 	files := []testtools.FileSpec{
 		{Path: "WORKSPACE"},
+		{Path: "BUILD.bazel"},
```

Comment:
This will create an empty `BUILD.bazel` file before running Gazelle. We want to verify the diff is empty if the file is *not* present though, so let's take this out.

Reply:
So, I originally had this backwards with what was erroring. I thought I'd checked my local code well enough, but apparently the issues raised in #925 were exactly the same as those in #853: the BUILD files existed and were empty.

That's why my initial test wouldn't fail with the original code, because the missing-file case was actually fine.

I've added two cases now, `TestDiffMissingAndNoChange` and `TestDiffEmptyAndNoChange`. The first passes both with the original code and my changes, and the second fails with the original code but passes with my changes.